### PR TITLE
Lookup: subscription by 'on' method should work correct for onPageLoading and onPullRefresh events (#13099)

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 1]
```diff
@@ -3467,6 +3467,89 @@ QUnit.module('Events', {
...
+                store: data,
+                paginate: true,
+                pageSize: 1
+            },
+            onPageLoading: (e) => {
+                assert.ok(true, 'onPageLoading is fired');
+            }
+        }).dxLookup('instance');
+
+        instance._list._scrollBottomHandler();
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]: Can we avoid using a private API here?

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]: fixed

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 2]
```diff
@@ -3467,6 +3467,99 @@ QUnit.module('Events', {
...
+                assert.ok(true, 'onPageLoading is fired');
+                done();
+            }
+        }).dxLookup('instance');
+
+        const list = instance._list;
+        list.option('useNative', true);
+        instance.open();
+
+        setTimeout(() => {
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]: Do we need this `setTimeout`?

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 3]
```diff
@@ -3467,6 +3467,89 @@ QUnit.module('Events', {
...
+                store: data,
+                paginate: true,
+                pageSize: 40
+            },
+            onPageLoading: (e) => {
+                assert.ok(true, 'onPageLoading is fired');
+            }
+        }).dxLookup('instance');
+
+        instance._list.option('onPageLoading')();
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]: Can we get a list instance using CSS selector?

**Review by San4es — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**